src/components/MultipieceModal/MultipieceModal.test.js

```diff
@@ -0,0 +1,239 @@
+import React from 'react';
+import {
+  fireEvent,
+  render,
+  screen,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+import {
+  componentPropsCheck,
+  getById,
+} from '../../../test/jest/helpers'
+
+import {
+  Modal,
+  ModalFooter,
+  Button,
+  KeyValue,
+} from '@folio/stripes/components';
+
+import MultipieceModal from './MultipieceModal';
+
+describe('MultipieceModal', () => {
+  const testIds = {
+    multipieceModal: 'multipieceModal',
+    multipieceModalFooter: 'multipieceModalFooter',
+    multipieceModalConfirmButton: 'multipieceModalConfirmButton',
+    multipieceModalCancelButton: 'multipieceModalCancelButton',
+    numberOfPieces: 'numberOfPieces',
+    descriptionOfPieces: 'descriptionOfPieces',
+    numberOfMissingPieces: 'numberOfMissingPieces',
+    descriptionOfmissingPieces: 'descriptionOfmissingPieces',
+  };
+  const labelIds = {
+    multipieceModalLabel: 'ui-checkin.multipieceModal.label',
+    multipieceModalCheckIn: 'ui-checkin.multipieceModal.checkIn',
+    multipieceModalCancel: 'ui-checkin.multipieceModal.cancel',
+    multipieceModalMessage: 'ui-checkin.multipieceModal.message',
+    itemNumberOfPieces: 'ui-checkin.multipieceModal.item.numberOfPieces',
+    itemDescriptionOfPieces: 'ui-checkin.multipieceModal.item.descriptionOfPieces',
+    itemNumberOfMissingPieces: 'ui-checkin.multipieceModal.item.numberOfMissingPieces',
+    itemDescriptionOfMissingPieces: 'ui-checkin.multipieceModal.item.descriptionOfmissingPieces',
+  };
+  const testOpen = false;
+  const testItem = {
+    title: 'testTitle',
+    barcode: 'testBarcode',
+    materialType: {
+      name: 'testName',
+    },
+  };
+  const testOnClose = jest.fn();
+  const testOnConfirm = jest.fn();
+  const testDefaultProps = {
+    onClose: testOnClose,
+    onConfirm: testOnConfirm,
+    item: testItem,
+    open: testOpen,
+  };
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('with default props', () => {
+    beforeEach(() => {
+      render(<MultipieceModal {...testDefaultProps} />);
+    });
+
+    it('should render multipiece Modal component', () => {
```

Comment:
Could you please wrap component names in double quotes?
It relates for whole test I think.

Reply:
done